motoman_gp200r_support/urdf/gp200r_macro.xacro

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" ?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+<xacro:macro name="motoman_gp200r" params="prefix">
+
+    <!-- Properties -->
+    <material name="yaskawa_blue">
+        <color rgba="0 0.35546875 0.671875 1" />
+    </material>
```

Comment:
Could you please use the `material` defs from `motoman_resources` instead?

Reply:
I have added a line in the urdf to load the xacro for common materials provided by motoman_resources so that ```"material_yaskawa_blue"``` is used.
```
<xacro:include filename="$(find motoman_resources)/urdf/common_materials.xacro"/>
```